.docker/Makfile

```diff
@@ -0,0 +1,16 @@
+all: help
```

Comment:
Note that you misspelled the file :) Makfile instead of Makefile

Reply:
Done